src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -192,7 +195,7 @@ public String getKey() {
     return key.get();
   }
 
-  public void setKey(String key) {
+  private void setKey(String key) {
```

Comment:
Just had a thought... Since we now have setters and getters for the key, it may make sense to get rid of the "key" parameter of each method entirely and base each method on taking the key which is currently set in the model. This would make it a bit easier because if we for example want to kill the connection, we can use "model.killConnection()" instead of "model.killConnection(model.getKey())", what do you think?